lib/configure.js

```diff
@@ -288,8 +287,9 @@ function configure (gyp, argv, callback) {
       argv.unshift(gyp_script)
 
       // make sure python uses files that came with this particular node package
-      var pypath = new PathArray(process.env, 'PYTHONPATH')
+      var pypath = process.env.PYTHONPATH ? [process.env.PYTHONPATH] : []
       pypath.unshift(path.join(__dirname, '..', 'gyp', 'pylib'))
+      process.env.PYTHONPATH = pypath.join(process.platform === 'win32' ? ';' : ':')
```

Comment:
I mean the second one.  I wrote `'PYTHONPATH' in process.env` but perhaps checking for truthiness / non-emptiness (i.e., just `process.env.PYTHONPATH`) is better.